mst/dblink-extension.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Using the `dblink` extension in Managed Service for TimescaleDB
+excerpt: Learn how to use `dblink` extension and connect to other PostgreSQL databases
+tags: [extension]
+---
+
+# Use the PostgreSQL `dblink` extension
+
+`dblink` is a [PostgreSQLextension](https://www.postgresql.org/docs/current/dblink.html)
+that allows you to connect to other PostgreSQL databases and to run arbitrary queries.
```

Comment:
```suggestion
The `dblink` [PostgreSQL extension](https://www.postgresql.org/docs/current/dblink.html)
allows you to connect to other PostgreSQL databases and to run arbitrary queries.
```

Just so that we're not starting the sentence with marked up text.